Modules/Microsoft365DSC/Examples/Resources/TeamsGroupPolicyAssignment/1-TeamsgrouppolicyAssignment.ps1

```diff
@@ -0,0 +1,28 @@
+<#
+This examples configure a TeamsGroupPolicyAssignment.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $credsGlobalAdmin
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    node localhost
+    {
+        TeamsGroupPolicyAssignment "TeamsGroupPolicyAssignment"
+        {
+            Id					  = "1"
+			Ensure                = "Present";
```

Comment:
Please make sure all parameters are aligned, have values that make sense and are valid. Examples are used in generating the documentation on the microsoft365dsc.com website, so must be as accurate and descriptive as possible.